samples/experimental/skills-prototypes/dialog-to-dialog/dialog-echo-skill-bot/config.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python3
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import os
+
+
+class DefaultConfig:
+    """ Bot Configuration """
+
+    PORT = 39793
+    APP_ID = os.environ.get("MicrosoftAppId", "fb7a9f3c-2b30-4ac8-86a0-c44bdeaa380e")
+    APP_PASSWORD = os.environ.get("MicrosoftAppPassword", "b0tframew0rks3cr3t!")
```

Comment:
bot appid and password should be removed.

Reply:
This is a work in progress.  It will.  Once it actually fully functions.